start-site/src/main/resources/application.yml

```diff
@@ -105,8 +105,15 @@ initializr:
         mappings:
           - compatibilityRange: "[2.0.0.RELEASE,2.0.x.BUILD-SNAPSHOT]"
             version: 2.0.3.RELEASE
-          - compatibilityRange: "2.1.0.RELEASE"
+          - compatibilityRange: "[2.1.0.RELEASE,2.1.10.RELEASE]"
             version: 2.1.4.RELEASE
+          - compatibilityRange: "[2.1.10.RELEASE,2.1.x.BUILD-SNAPSHOT]"
+            version: 3.0.1.RELEASE
+          - compatibilityRange: "[2.1.x.BUILD-SNAPSHOT,2.2.0.M4]"
+            version: 3.0.2.BUILD-SNAPSHOT
```

Comment:
I would avoid the snapshot for this. We usually enable that for the next feature release only.